{"time": "2025-03-27 19:07:57,171", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 19:08:22,246", "level": "WARNING", "message": "Failed login attempt for svkhsj from 127.0.0.1"}
{"time": "2025-03-27 19:08:22,246", "level": "WARNING", "message": "Failed login attempt for username: svkhsj"}
{"time": "2025-03-27 19:08:22,262", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-27 19:10:23,619", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\apps.py changed, reloading."}
{"time": "2025-03-27 19:10:25,329", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 19:10:26,690", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\apps.py changed, reloading."}
{"time": "2025-03-27 19:10:27,805", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 19:10:31,320", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\apps.py changed, reloading."}
{"time": "2025-03-27 19:10:32,523", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 19:10:41,294", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\apps.py changed, reloading."}
{"time": "2025-03-27 19:10:42,595", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 19:10:47,160", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\apps.py changed, reloading."}
{"time": "2025-03-27 19:10:48,247", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 19:10:53,709", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\apps.py changed, reloading."}
{"time": "2025-03-27 19:10:55,037", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 19:13:30,182", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 19:13:33,898", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-03-27 19:13:33,899", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-03-27 19:13:38,264", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-27 19:13:44,495", "level": "INFO", "message": "User subham logged in at 2025-03-27 13:43:44.495470+00:00 from IP 127.0.0.1"}
{"time": "2025-03-27 19:13:44,495", "level": "INFO", "message": "User subham logged in from 127.0.0.1"}
{"time": "2025-03-27 19:13:44,503", "level": "INFO", "message": "User subham logged in"}
{"time": "2025-03-27 19:13:44,503", "level": "INFO", "message": "User subham logged in successfully."}
{"time": "2025-03-27 19:13:44,506", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-03-27 19:13:44,521", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-03-27 19:13:45,675", "level": "INFO", "message": "User subham logged out at 2025-03-27 13:43:45.675275+00:00 from IP 127.0.0.1"}
{"time": "2025-03-27 19:13:45,675", "level": "INFO", "message": "User subham logged out"}
{"time": "2025-03-27 19:13:45,695", "level": "INFO", "message": "User subham logged out successfully."}
{"time": "2025-03-27 19:13:45,695", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-03-27 19:13:45,695", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-27 21:34:34,663", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 21:35:32,874", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\apps.py changed, reloading."}
{"time": "2025-03-27 21:35:33,929", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 21:36:00,328", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 21:37:35,570", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 21:38:59,755", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 21:40:01,424", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-27 21:40:06,507", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-03-27 21:40:06,507", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-03-27 21:40:11,968", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-27 21:40:16,316", "level": "WARNING", "message": "Failed login attempt for dcsdv from 127.0.0.1"}
{"time": "2025-03-27 21:40:16,330", "level": "WARNING", "message": "Failed login attempt for username: dcsdv"}
{"time": "2025-03-27 21:40:16,332", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-28 10:05:24,136", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:06:29,219", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:06:53,323", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-28 10:06:55,789", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:07:00,146", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-28 10:07:03,067", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:07:13,409", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:08:13,074", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:08:33,810", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:09:29,600", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:09:53,304", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-28 10:09:57,864", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:10:13,162", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-28 10:10:47,640", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:13:49,492", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:17:01,479", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:19:39,610", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:20:00,823", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:27:41,017", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:32:24,035", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:38:32,599", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:42:59,661", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:44:52,195", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:45:28,989", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:46:31,166", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-28 10:46:31,419", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-28 10:46:33,974", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:46:35,601", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-28 10:46:35,735", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:47:59,181", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-28 10:48:02,564", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:48:24,626", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:51:06,987", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:52:06,385", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:52:51,257", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-28 10:52:56,061", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:53:12,630", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:54:41,232", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-28 10:54:45,012", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:55:05,207", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-28 10:55:08,453", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:55:25,981", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 10:55:34,201", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-03-28 10:55:34,201", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-03-28 10:55:34,502", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-03-28 10:55:34,504", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 2356"}
{"time": "2025-03-28 10:55:37,737", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-28 10:55:41,938", "level": "WARNING", "message": "Failed login attempt for sdkhfa from 127.0.0.1"}
{"time": "2025-03-28 10:55:41,938", "level": "WARNING", "message": "Failed login attempt for username: sdkhfa"}
{"time": "2025-03-28 10:55:41,940", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-28 10:55:46,493", "level": "WARNING", "message": "Failed login attempt for sdjlvskvgn from 127.0.0.1"}
{"time": "2025-03-28 10:55:46,493", "level": "WARNING", "message": "Failed login attempt for username: sdjlvskvgn"}
{"time": "2025-03-28 10:55:46,493", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-28 11:06:18,932", "level": "WARNING", "message": "Failed login attempt for hhmfgnmvhm from 127.0.0.1"}
{"time": "2025-03-28 11:06:18,970", "level": "WARNING", "message": "Failed login attempt for username: hhmfgnmvhm"}
{"time": "2025-03-28 11:06:18,976", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-28 21:30:00,218", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:30:10,201", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-03-28 21:30:10,201", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-03-28 21:30:10,310", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-03-28 21:30:10,311", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 2356"}
{"time": "2025-03-28 21:30:14,990", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-28 21:30:21,391", "level": "WARNING", "message": "Failed login attempt for dcscv from 127.0.0.1"}
{"time": "2025-03-28 21:30:21,393", "level": "WARNING", "message": "Failed login attempt for username: dcscv"}
{"time": "2025-03-28 21:30:21,393", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-28 21:30:28,631", "level": "WARNING", "message": "Failed login attempt for fvadsfr from 127.0.0.1"}
{"time": "2025-03-28 21:30:28,631", "level": "WARNING", "message": "Failed login attempt for username: fvadsfr"}
{"time": "2025-03-28 21:30:28,633", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-28 21:36:54,186", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:37:14,247", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-28 21:37:14,247", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 2380"}
{"time": "2025-03-28 21:38:36,916", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\urls.py changed, reloading."}
{"time": "2025-03-28 21:38:40,155", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:38:46,952", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\urls.py changed, reloading."}
{"time": "2025-03-28 21:38:50,102", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:38:56,850", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\urls.py changed, reloading."}
{"time": "2025-03-28 21:39:00,217", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:39:12,316", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:39:26,304", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\urls.py changed, reloading."}
{"time": "2025-03-28 21:39:29,843", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:39:36,515", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\urls.py changed, reloading."}
{"time": "2025-03-28 21:39:40,183", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:39:54,809", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:40:08,804", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\urls.py changed, reloading."}
{"time": "2025-03-28 21:40:08,871", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\urls.py changed, reloading."}
{"time": "2025-03-28 21:40:13,791", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:40:14,011", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:40:31,183", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:40:51,715", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-28 21:40:51,715", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 27"}
{"time": "2025-03-28 21:41:25,188", "level": "INFO", "message": ""GET /admin/ HTTP/1.1" 302 0"}
{"time": "2025-03-28 21:41:25,585", "level": "INFO", "message": ""GET /admin/login/?next=/admin/ HTTP/1.1" 200 4158"}
{"time": "2025-03-28 21:41:26,554", "level": "INFO", "message": ""GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2682"}
{"time": "2025-03-28 21:41:26,584", "level": "INFO", "message": ""GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810"}
{"time": "2025-03-28 21:41:26,599", "level": "INFO", "message": ""GET /static/admin/css/base.css HTTP/1.1" 200 21544"}
{"time": "2025-03-28 21:41:26,603", "level": "INFO", "message": ""GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063"}
{"time": "2025-03-28 21:41:26,609", "level": "INFO", "message": ""GET /static/admin/css/responsive.css HTTP/1.1" 200 17905"}
{"time": "2025-03-28 21:41:26,616", "level": "INFO", "message": ""GET /static/admin/css/login.css HTTP/1.1" 200 958"}
{"time": "2025-03-28 21:41:26,644", "level": "INFO", "message": ""GET /static/admin/js/theme.js HTTP/1.1" 200 1943"}
{"time": "2025-03-28 21:41:33,866", "level": "INFO", "message": "User subham logged in from 127.0.0.1"}
{"time": "2025-03-28 21:41:33,868", "level": "INFO", "message": ""POST /admin/login/?next=/admin/ HTTP/1.1" 302 0"}
{"time": "2025-03-28 21:41:33,914", "level": "INFO", "message": ""GET /admin/ HTTP/1.1" 200 5534"}
{"time": "2025-03-28 21:41:33,960", "level": "INFO", "message": ""GET /static/admin/css/dashboard.css HTTP/1.1" 200 441"}
{"time": "2025-03-28 21:41:33,976", "level": "INFO", "message": ""GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380"}
{"time": "2025-03-28 21:41:33,990", "level": "INFO", "message": ""GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331"}
{"time": "2025-03-28 21:41:36,134", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 200 10761"}
{"time": "2025-03-28 21:41:36,167", "level": "INFO", "message": ""GET /static/admin/css/changelists.css HTTP/1.1" 200 6811"}
{"time": "2025-03-28 21:41:36,179", "level": "INFO", "message": ""GET /static/admin/js/jquery.init.js HTTP/1.1" 200 347"}
{"time": "2025-03-28 21:41:36,217", "level": "INFO", "message": ""GET /static/admin/js/actions.js HTTP/1.1" 200 8067"}
{"time": "2025-03-28 21:41:36,225", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-28 21:41:36,231", "level": "INFO", "message": ""GET /static/admin/js/urlify.js HTTP/1.1" 200 7887"}
{"time": "2025-03-28 21:41:36,242", "level": "INFO", "message": ""GET /static/admin/js/core.js HTTP/1.1" 200 6208"}
{"time": "2025-03-28 21:41:36,246", "level": "INFO", "message": ""GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 9042"}
{"time": "2025-03-28 21:41:36,256", "level": "INFO", "message": ""GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1531"}
{"time": "2025-03-28 21:41:36,279", "level": "INFO", "message": ""GET /static/admin/img/search.svg HTTP/1.1" 200 458"}
{"time": "2025-03-28 21:41:36,287", "level": "INFO", "message": ""GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436"}
{"time": "2025-03-28 21:41:36,321", "level": "INFO", "message": ""GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 285314"}
{"time": "2025-03-28 21:41:36,369", "level": "INFO", "message": ""GET /static/admin/js/filters.js HTTP/1.1" 200 978"}
{"time": "2025-03-28 21:41:36,440", "level": "INFO", "message": ""GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 325171"}
{"time": "2025-03-28 21:41:36,472", "level": "INFO", "message": ""GET /static/admin/img/sorting-icons.svg HTTP/1.1" 200 1097"}
{"time": "2025-03-28 21:41:36,481", "level": "INFO", "message": ""GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581"}
{"time": "2025-03-28 21:41:36,483", "level": "INFO", "message": ""GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331"}
{"time": "2025-03-28 21:42:21,239", "level": "ERROR", "message": "Internal Server Error: /api/get_email/subham/"}
Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\sqlite3\base.py", line 329, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such table: accounts_userprofile

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py", line 36, in get_user_email
    profile = UserProfile.objects.get(user=user)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 645, in get
    num = len(clone)
          ^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 382, in __len__
    self._fetch_all()
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\sqlite3\base.py", line 329, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such table: accounts_userprofile
{"time": "2025-03-28 21:42:21,250", "level": "ERROR", "message": ""GET /api/get_email/subham/ HTTP/1.1" 500 144718"}
{"time": "2025-03-28 21:42:44,537", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 200 10761"}
{"time": "2025-03-28 21:42:44,553", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-28 21:43:02,722", "level": "INFO", "message": ""GET /admin/auth/user/add/ HTTP/1.1" 200 10135"}
{"time": "2025-03-28 21:43:02,759", "level": "INFO", "message": ""GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 586"}
{"time": "2025-03-28 21:43:02,762", "level": "INFO", "message": ""GET /static/admin/css/forms.css HTTP/1.1" 200 9090"}
{"time": "2025-03-28 21:43:02,772", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-28 21:43:02,778", "level": "INFO", "message": ""GET /static/admin/css/widgets.css HTTP/1.1" 200 11800"}
{"time": "2025-03-28 21:43:02,781", "level": "INFO", "message": ""GET /static/admin/js/change_form.js HTTP/1.1" 200 606"}
{"time": "2025-03-28 21:43:27,271", "level": "INFO", "message": ""POST /admin/auth/user/add/ HTTP/1.1" 302 0"}
{"time": "2025-03-28 21:43:27,406", "level": "INFO", "message": ""GET /admin/auth/user/2/change/ HTTP/1.1" 200 20626"}
{"time": "2025-03-28 21:43:27,451", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-28 21:43:27,467", "level": "INFO", "message": ""GET /static/admin/js/calendar.js HTTP/1.1" 200 9141"}
{"time": "2025-03-28 21:43:27,483", "level": "INFO", "message": ""GET /static/admin/js/SelectFilter2.js HTTP/1.1" 200 15492"}
{"time": "2025-03-28 21:43:27,488", "level": "INFO", "message": ""GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 200 19319"}
{"time": "2025-03-28 21:43:27,496", "level": "INFO", "message": ""GET /static/admin/js/SelectBox.js HTTP/1.1" 200 4530"}
{"time": "2025-03-28 21:43:27,560", "level": "INFO", "message": ""GET /static/admin/img/icon-unknown.svg HTTP/1.1" 200 655"}
{"time": "2025-03-28 21:43:27,564", "level": "INFO", "message": ""GET /static/admin/img/icon-calendar.svg HTTP/1.1" 200 1086"}
{"time": "2025-03-28 21:43:27,570", "level": "INFO", "message": ""GET /static/admin/img/icon-clock.svg HTTP/1.1" 200 677"}
{"time": "2025-03-28 21:43:27,570", "level": "INFO", "message": ""GET /static/admin/img/selector-icons.svg HTTP/1.1" 200 3291"}
{"time": "2025-03-28 21:43:27,573", "level": "INFO", "message": ""GET /static/admin/img/icon-unknown-alt.svg HTTP/1.1" 200 655"}
{"time": "2025-03-28 21:43:35,309", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 200 11197"}
{"time": "2025-03-28 21:43:35,339", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-28 21:43:35,344", "level": "INFO", "message": ""GET /static/admin/img/icon-no.svg HTTP/1.1" 200 560"}
{"time": "2025-03-28 21:44:46,069", "level": "ERROR", "message": "Internal Server Error: /api/get_email/subham/"}
Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\sqlite3\base.py", line 329, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such table: accounts_userprofile

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py", line 36, in get_user_email
    profile = UserProfile.objects.get(user=user)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 645, in get
    num = len(clone)
          ^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 382, in __len__
    self._fetch_all()
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\sqlite3\base.py", line 329, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such table: accounts_userprofile
{"time": "2025-03-28 21:44:46,069", "level": "ERROR", "message": ""GET /api/get_email/subham/ HTTP/1.1" 500 144718"}
{"time": "2025-03-28 21:45:26,534", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 200 11197"}
{"time": "2025-03-28 21:45:26,595", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-28 21:45:28,619", "level": "INFO", "message": ""GET /admin/auth/user/1/change/ HTTP/1.1" 200 20503"}
{"time": "2025-03-28 21:45:28,632", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-28 21:45:34,670", "level": "INFO", "message": ""POST /admin/auth/user/1/change/ HTTP/1.1" 200 20676"}
{"time": "2025-03-28 21:45:34,673", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-28 21:45:41,525", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 200 11197"}
{"time": "2025-03-28 21:45:41,540", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-28 21:45:46,287", "level": "INFO", "message": ""GET /admin/auth/user/2/change/ HTTP/1.1" 200 20407"}
{"time": "2025-03-28 21:45:46,377", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-28 21:45:46,817", "level": "INFO", "message": ""GET /admin/auth/user/2/change/ HTTP/1.1" 200 20407"}
{"time": "2025-03-28 21:45:46,822", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-28 21:45:48,900", "level": "ERROR", "message": "Internal Server Error: /admin/auth/user/2/delete/"}
Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\sqlite3\base.py", line 329, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such table: accounts_userprofile

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 715, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\sites.py", line 240, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 2164, in delete_view
    return self._delete_view(request, object_id, extra_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 2191, in _delete_view
    ) = self.get_deleted_objects([obj], request)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 2159, in get_deleted_objects
    return get_deleted_objects(objs, request, self.admin_site)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\utils.py", line 137, in get_deleted_objects
    collector.collect(objs)
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\utils.py", line 204, in collect
    return super().collect(objs, source_attr=source_attr, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\deletion.py", line 348, in collect
    if getattr(on_delete, "lazy_sub_objs", False) or sub_objs:
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 414, in __bool__
    self._fetch_all()
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\sqlite3\base.py", line 329, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such table: accounts_userprofile
{"time": "2025-03-28 21:45:48,900", "level": "ERROR", "message": ""GET /admin/auth/user/2/delete/ HTTP/1.1" 500 211909"}
{"time": "2025-03-28 21:45:53,505", "level": "INFO", "message": ""GET /admin/auth/user/2/change/ HTTP/1.1" 200 20407"}
{"time": "2025-03-28 21:45:55,378", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 200 11197"}
{"time": "2025-03-28 21:46:04,149", "level": "INFO", "message": ""GET /admin/auth/user/2/change/ HTTP/1.1" 200 20407"}
{"time": "2025-03-28 21:46:04,171", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-28 21:46:07,670", "level": "ERROR", "message": "Internal Server Error: /admin/auth/user/2/delete/"}
Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\sqlite3\base.py", line 329, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
sqlite3.OperationalError: no such table: accounts_userprofile

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 715, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\sites.py", line 240, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 2164, in delete_view
    return self._delete_view(request, object_id, extra_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 2191, in _delete_view
    ) = self.get_deleted_objects([obj], request)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 2159, in get_deleted_objects
    return get_deleted_objects(objs, request, self.admin_site)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\utils.py", line 137, in get_deleted_objects
    collector.collect(objs)
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\utils.py", line 204, in collect
    return super().collect(objs, source_attr=source_attr, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\deletion.py", line 348, in collect
    if getattr(on_delete, "lazy_sub_objs", False) or sub_objs:
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 414, in __bool__
    self._fetch_all()
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 1928, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\db\backends\sqlite3\base.py", line 329, in execute
    return super().execute(query, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.OperationalError: no such table: accounts_userprofile
{"time": "2025-03-28 21:46:07,677", "level": "ERROR", "message": ""GET /admin/auth/user/2/delete/ HTTP/1.1" 500 211909"}
{"time": "2025-03-28 21:46:16,177", "level": "INFO", "message": ""GET /admin/auth/user/2/change/ HTTP/1.1" 200 20407"}
{"time": "2025-03-28 21:46:51,009", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-03-28 21:46:51,354", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-03-28 21:46:53,314", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:46:53,968", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:47:00,389", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-03-28 21:47:00,773", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-03-28 21:47:04,473", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:47:04,484", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:47:07,883", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-03-28 21:47:07,907", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-03-28 21:47:11,335", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:47:13,914", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-03-28 21:47:14,220", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:47:15,648", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-03-28 21:47:25,708", "level": "WARNING", "message": "Not Found: /api/get_email/jiyachaudhari0703@gmail.com/"}
{"time": "2025-03-28 21:47:25,708", "level": "WARNING", "message": ""GET /api/get_email/jiyachaudhari0703@gmail.com/ HTTP/1.1" 404 27"}
{"time": "2025-03-28 21:55:19,296", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-28 21:55:46,368", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-28 21:55:46,370", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 27"}
{"time": "2025-03-28 21:56:01,635", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-28 21:56:01,635", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 27"}
{"time": "2025-03-28 21:56:10,518", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-28 21:56:10,518", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 27"}
{"time": "2025-03-29 18:11:49,403", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:11:58,227", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-03-29 18:11:58,229", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 3006"}
{"time": "2025-03-29 18:11:58,375", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-03-29 18:11:58,377", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 3057"}
{"time": "2025-03-29 18:12:02,076", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-29 18:12:19,070", "level": "WARNING", "message": "Failed login attempt for jiyachaudhari0703@gmail.com from 127.0.0.1"}
{"time": "2025-03-29 18:12:19,072", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari0703@gmail.com"}
{"time": "2025-03-29 18:12:19,076", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-29 18:12:36,572", "level": "WARNING", "message": "Failed login attempt for jiyachaudhari0703@gmail.com from 127.0.0.1"}
{"time": "2025-03-29 18:12:36,572", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari0703@gmail.com"}
{"time": "2025-03-29 18:12:36,576", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-29 18:18:46,925", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-29 18:18:46,925", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 27"}
{"time": "2025-03-29 18:23:59,042", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:24:06,994", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-03-29 18:24:06,997", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 3006"}
{"time": "2025-03-29 18:24:26,076", "level": "WARNING", "message": "Not Found: /api/get_email/subham/"}
{"time": "2025-03-29 18:24:26,076", "level": "WARNING", "message": ""GET /api/get_email/subham/ HTTP/1.1" 404 27"}
{"time": "2025-03-29 18:24:33,203", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-29 18:24:33,205", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 27"}
{"time": "2025-03-29 18:24:41,550", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 200 11197"}
{"time": "2025-03-29 18:24:41,614", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-29 18:25:43,719", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 200 11197"}
{"time": "2025-03-29 18:25:43,733", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-29 18:26:13,684", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-29 18:26:13,688", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 27"}
{"time": "2025-03-29 18:26:16,009", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-29 18:26:16,009", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 27"}
{"time": "2025-03-29 18:26:16,256", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-29 18:26:16,256", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 27"}
{"time": "2025-03-29 18:26:16,431", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-29 18:26:16,431", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 27"}
{"time": "2025-03-29 18:26:16,597", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-29 18:26:16,599", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 27"}
{"time": "2025-03-29 18:26:16,828", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-29 18:26:16,828", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 27"}
{"time": "2025-03-29 18:26:17,026", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-29 18:26:17,042", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 27"}
{"time": "2025-03-29 18:29:24,797", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/
"}
{"time": "2025-03-29 18:29:24,800", "level": "WARNING", "message": ""GET /api/get_email/jiya/%0A HTTP/1.1" 404 3086"}
{"time": "2025-03-29 18:29:31,684", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/
"}
{"time": "2025-03-29 18:29:31,686", "level": "WARNING", "message": ""GET /api/get_email/jiya/%0A HTTP/1.1" 404 3086"}
{"time": "2025-03-29 18:29:34,593", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/
"}
{"time": "2025-03-29 18:29:34,593", "level": "WARNING", "message": ""GET /api/get_email/jiya/%0A HTTP/1.1" 404 3086"}
{"time": "2025-03-29 18:33:03,971", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\urls.py changed, reloading."}
{"time": "2025-03-29 18:33:08,316", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:33:36,881", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\urls.py changed, reloading."}
{"time": "2025-03-29 18:33:40,426", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:33:41,947", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\urls.py changed, reloading."}
{"time": "2025-03-29 18:33:45,759", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:34:21,698", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-03-29 18:34:24,324", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:35:41,822", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-03-29 18:35:45,750", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:36:08,367", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:36:16,108", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-03-29 18:36:16,110", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-03-29 18:36:23,932", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-29 18:36:37,697", "level": "WARNING", "message": "Failed login attempt for jiyachaudhari from 127.0.0.1."}
{"time": "2025-03-29 18:36:37,697", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari from 127.0.0.1"}
{"time": "2025-03-29 18:36:37,697", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-29 18:36:50,479", "level": "WARNING", "message": "Not Found: /accounts/api/get_email/jiya/

"}
{"time": "2025-03-29 18:36:50,489", "level": "WARNING", "message": ""GET /accounts/api/get_email/jiya/%0A%0A HTTP/1.1" 404 3045"}
{"time": "2025-03-29 18:36:52,537", "level": "WARNING", "message": "Not Found: /accounts/api/get_email/jiya/

"}
{"time": "2025-03-29 18:36:52,537", "level": "WARNING", "message": ""GET /accounts/api/get_email/jiya/%0A%0A HTTP/1.1" 404 3045"}
{"time": "2025-03-29 18:37:12,629", "level": "WARNING", "message": "Not Found: /accounts/api/get_email/jiya/
"}
{"time": "2025-03-29 18:37:12,629", "level": "WARNING", "message": ""GET /accounts/api/get_email/jiya/%0A HTTP/1.1" 404 3040"}
{"time": "2025-03-29 18:38:23,717", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-03-29 18:38:27,089", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:38:41,428", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\models.py changed, reloading."}
{"time": "2025-03-29 18:38:45,455", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:42:12,034", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 200 11197"}
{"time": "2025-03-29 18:42:12,086", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-29 18:42:20,354", "level": "INFO", "message": ""GET /admin/auth/user/2/change/ HTTP/1.1" 200 20762"}
{"time": "2025-03-29 18:42:20,371", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-29 18:42:27,061", "level": "INFO", "message": ""POST /admin/auth/user/2/change/ HTTP/1.1" 302 0"}
{"time": "2025-03-29 18:42:27,098", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 200 11389"}
{"time": "2025-03-29 18:42:27,124", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-29 18:42:30,470", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 200 11197"}
{"time": "2025-03-29 18:42:30,487", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-29 18:42:31,343", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 200 11197"}
{"time": "2025-03-29 18:42:31,371", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-29 18:43:47,429", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 200 11197"}
{"time": "2025-03-29 18:43:47,445", "level": "INFO", "message": ""GET /admin/jsi18n/ HTTP/1.1" 200 3342"}
{"time": "2025-03-29 18:44:09,160", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:44:21,067", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-03-29 18:44:21,068", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-03-29 18:44:37,433", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-29 18:45:12,158", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-29 18:45:12,158", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 2380"}
{"time": "2025-03-29 18:45:22,402", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/
"}
{"time": "2025-03-29 18:45:22,402", "level": "WARNING", "message": ""GET /api/get_email/jiya/%0A HTTP/1.1" 404 2385"}
{"time": "2025-03-29 18:45:54,779", "level": "WARNING", "message": "Not Found: /accounts/get_email/jiya/"}
{"time": "2025-03-29 18:45:54,781", "level": "WARNING", "message": ""GET /accounts/get_email/jiya/ HTTP/1.1" 404 3023"}
{"time": "2025-03-29 18:47:10,672", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\urls.py changed, reloading."}
{"time": "2025-03-29 18:47:13,280", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:47:14,803", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\urls.py changed, reloading."}
{"time": "2025-03-29 18:47:28,022", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-29 18:47:28,022", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 3097"}
{"time": "2025-03-29 18:47:31,005", "level": "WARNING", "message": "Not Found: /api/get_email/jiya/"}
{"time": "2025-03-29 18:47:31,005", "level": "WARNING", "message": ""GET /api/get_email/jiya/ HTTP/1.1" 404 3097"}
{"time": "2025-03-29 18:48:02,291", "level": "WARNING", "message": "Not Found: /accounts/api/get_email/jiya/"}
{"time": "2025-03-29 18:48:02,291", "level": "WARNING", "message": ""GET /accounts/api/get_email/jiya/ HTTP/1.1" 404 3068"}
{"time": "2025-03-29 18:48:19,962", "level": "WARNING", "message": "Not Found: /accounts/api/get_email/jiya/"}
{"time": "2025-03-29 18:48:19,962", "level": "WARNING", "message": ""GET /accounts/api/get_email/jiya/ HTTP/1.1" 404 3068"}
{"time": "2025-03-29 18:48:40,598", "level": "WARNING", "message": "Not Found: /accounts/api/get_email/jiya/"}
{"time": "2025-03-29 18:48:40,601", "level": "WARNING", "message": ""GET /accounts/api/get_email/jiya/ HTTP/1.1" 404 3068"}
{"time": "2025-03-29 18:48:40,708", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-03-29 18:48:40,711", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 2460"}
{"time": "2025-03-29 18:49:48,630", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:49:56,787", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-03-29 18:49:56,795", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2300"}
{"time": "2025-03-29 18:50:04,768", "level": "WARNING", "message": "Not Found: /api"}
{"time": "2025-03-29 18:50:04,774", "level": "WARNING", "message": ""GET /api HTTP/1.1" 404 2327"}
{"time": "2025-03-29 18:50:17,146", "level": "WARNING", "message": "Not Found: /accounts/api/"}
{"time": "2025-03-29 18:50:17,146", "level": "WARNING", "message": ""GET /accounts/api/ HTTP/1.1" 404 2357"}
{"time": "2025-03-29 18:50:28,609", "level": "WARNING", "message": "Not Found: /accounts/api/get_email/jiya/"}
{"time": "2025-03-29 18:50:28,609", "level": "WARNING", "message": ""GET /accounts/api/get_email/jiya/ HTTP/1.1" 404 2402"}
{"time": "2025-03-29 18:50:32,888", "level": "WARNING", "message": "Not Found: /accounts/api/get_email/jiya/"}
{"time": "2025-03-29 18:50:32,888", "level": "WARNING", "message": ""GET /accounts/api/get_email/jiya/ HTTP/1.1" 404 2402"}
{"time": "2025-03-29 18:53:35,591", "level": "WARNING", "message": "Not Found: /accounts/api/get_email/newjiya/"}
{"time": "2025-03-29 18:53:35,591", "level": "WARNING", "message": ""GET /accounts/api/get_email/newjiya/ HTTP/1.1" 404 2411"}
{"time": "2025-03-29 18:54:16,774", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\urls.py changed, reloading."}
{"time": "2025-03-29 18:54:19,424", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 18:54:20,974", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\urls.py changed, reloading."}
{"time": "2025-03-29 18:54:27,641", "level": "INFO", "message": ""GET /accounts/api/get_email/newjiya/ HTTP/1.1" 200 32"}
{"time": "2025-03-29 18:54:49,957", "level": "INFO", "message": ""GET /accounts/api/get_email/newjiya/ HTTP/1.1" 200 32"}
{"time": "2025-03-29 21:44:26,706", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 21:44:47,982", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-03-29 21:44:47,982", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2409"}
{"time": "2025-03-29 21:45:16,715", "level": "WARNING", "message": "Not Found: /send-email/"}
{"time": "2025-03-29 21:45:16,724", "level": "WARNING", "message": ""GET /send-email/ HTTP/1.1" 404 2460"}
{"time": "2025-03-29 21:46:17,645", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\urls.py changed, reloading."}
{"time": "2025-03-29 21:46:20,748", "level": "WARNING", "message": "Not Found: /send-email/"}
{"time": "2025-03-29 21:46:20,748", "level": "WARNING", "message": ""GET /send-email/ HTTP/1.1" 404 2460"}
{"time": "2025-03-29 21:46:21,921", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 21:46:24,711", "level": "WARNING", "message": "Not Found: /send-email/"}
{"time": "2025-03-29 21:46:24,711", "level": "WARNING", "message": ""GET /send-email/ HTTP/1.1" 404 2460"}
{"time": "2025-03-29 21:46:36,265", "level": "WARNING", "message": "Not Found: /accounts/send-email/"}
{"time": "2025-03-29 21:46:36,265", "level": "WARNING", "message": ""GET /accounts/send-email/ HTTP/1.1" 404 3115"}
{"time": "2025-03-29 21:47:10,918", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\urls.py changed, reloading."}
{"time": "2025-03-29 21:47:14,172", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 21:47:38,212", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 21:48:40,008", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\urls.py changed, reloading."}
{"time": "2025-03-29 21:48:43,724", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 21:49:59,420", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 21:50:15,647", "level": "WARNING", "message": "Method Not Allowed: /accounts/send-email/"}
{"time": "2025-03-29 21:50:15,647", "level": "WARNING", "message": ""GET /accounts/send-email/ HTTP/1.1" 405 35"}
{"time": "2025-03-29 21:51:00,814", "level": "WARNING", "message": "Forbidden (CSRF cookie not set.): /accounts/send-email/"}
{"time": "2025-03-29 21:51:00,817", "level": "WARNING", "message": ""POST /accounts/send-email/ HTTP/1.1" 403 2869"}
{"time": "2025-03-29 21:51:31,721", "level": "WARNING", "message": "Forbidden (CSRF cookie not set.): /accounts/send-email/"}
{"time": "2025-03-29 21:51:31,721", "level": "WARNING", "message": ""POST /accounts/send-email/ HTTP/1.1" 403 2869"}
{"time": "2025-03-29 21:51:38,351", "level": "WARNING", "message": "Forbidden (CSRF cookie not set.): /accounts/send-email/"}
{"time": "2025-03-29 21:51:38,351", "level": "WARNING", "message": ""POST /accounts/send-email/ HTTP/1.1" 403 2869"}
{"time": "2025-03-29 21:52:03,776", "level": "WARNING", "message": "Forbidden (CSRF cookie not set.): /accounts/send-email/"}
{"time": "2025-03-29 21:52:03,776", "level": "WARNING", "message": ""POST /accounts/send-email/ HTTP/1.1" 403 2869"}
{"time": "2025-03-29 21:55:57,680", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-03-29 21:56:01,099", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 21:56:20,549", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\urls.py changed, reloading."}
{"time": "2025-03-29 21:56:25,179", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 21:56:29,950", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\urls.py changed, reloading."}
{"time": "2025-03-29 21:56:33,382", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 21:56:46,882", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 21:57:03,118", "level": "WARNING", "message": "Not Found: /send-email/"}
{"time": "2025-03-29 21:57:03,118", "level": "WARNING", "message": ""GET /send-email/ HTTP/1.1" 404 2356"}
{"time": "2025-03-29 21:57:32,563", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\urls.py changed, reloading."}
{"time": "2025-03-29 21:57:35,146", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 21:57:57,150", "level": "WARNING", "message": "Bad Request: /accounts/send-email/"}
{"time": "2025-03-29 21:57:57,152", "level": "WARNING", "message": ""GET /accounts/send-email/ HTTP/1.1" 400 35"}
{"time": "2025-03-29 21:59:10,046", "level": "WARNING", "message": "Forbidden (CSRF cookie not set.): /accounts/send-email/"}
{"time": "2025-03-29 21:59:10,050", "level": "WARNING", "message": ""POST /accounts/send-email/ HTTP/1.1" 403 2869"}
{"time": "2025-03-29 21:59:41,676", "level": "WARNING", "message": "Not Found: /send-email/"}
{"time": "2025-03-29 21:59:41,684", "level": "WARNING", "message": ""GET /send-email/ HTTP/1.1" 404 2356"}
{"time": "2025-03-29 22:00:03,154", "level": "WARNING", "message": "Forbidden (CSRF cookie not set.): /accounts/send-email/"}
{"time": "2025-03-29 22:00:03,154", "level": "WARNING", "message": ""POST /accounts/send-email/ HTTP/1.1" 403 2869"}
{"time": "2025-03-29 22:00:43,823", "level": "WARNING", "message": "Bad Request: /accounts/send-email/"}
{"time": "2025-03-29 22:00:43,823", "level": "WARNING", "message": ""GET /accounts/send-email/ HTTP/1.1" 400 35"}
{"time": "2025-03-29 22:01:42,697", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-03-29 22:01:45,612", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 22:04:00,587", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 22:04:19,381", "level": "WARNING", "message": "Method Not Allowed: /accounts/send-email/"}
{"time": "2025-03-29 22:04:19,385", "level": "WARNING", "message": ""GET /accounts/send-email/ HTTP/1.1" 405 35"}
{"time": "2025-03-29 22:04:43,900", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 22:04:58,656", "level": "WARNING", "message": "Method Not Allowed: /accounts/send-email/"}
{"time": "2025-03-29 22:04:58,659", "level": "WARNING", "message": ""GET /accounts/send-email/ HTTP/1.1" 405 35"}
{"time": "2025-03-29 22:04:58,733", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-03-29 22:04:58,735", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 2356"}
{"time": "2025-03-29 22:05:01,216", "level": "WARNING", "message": "Method Not Allowed: /accounts/send-email/"}
{"time": "2025-03-29 22:05:01,219", "level": "WARNING", "message": ""GET /accounts/send-email/ HTTP/1.1" 405 35"}
{"time": "2025-03-29 22:06:57,416", "level": "WARNING", "message": "Method Not Allowed: /accounts/send-email/"}
{"time": "2025-03-29 22:06:57,429", "level": "WARNING", "message": ""GET /accounts/send-email/ HTTP/1.1" 405 35"}
{"time": "2025-03-29 22:06:59,197", "level": "WARNING", "message": "Method Not Allowed: /accounts/send-email/"}
{"time": "2025-03-29 22:06:59,197", "level": "WARNING", "message": ""GET /accounts/send-email/ HTTP/1.1" 405 35"}
{"time": "2025-03-29 22:06:59,386", "level": "WARNING", "message": "Method Not Allowed: /accounts/send-email/"}
{"time": "2025-03-29 22:06:59,389", "level": "WARNING", "message": ""GET /accounts/send-email/ HTTP/1.1" 405 35"}
{"time": "2025-03-29 22:06:59,564", "level": "WARNING", "message": "Method Not Allowed: /accounts/send-email/"}
{"time": "2025-03-29 22:06:59,566", "level": "WARNING", "message": ""GET /accounts/send-email/ HTTP/1.1" 405 35"}
{"time": "2025-03-29 22:06:59,732", "level": "WARNING", "message": "Method Not Allowed: /accounts/send-email/"}
{"time": "2025-03-29 22:06:59,733", "level": "WARNING", "message": ""GET /accounts/send-email/ HTTP/1.1" 405 35"}
{"time": "2025-03-29 22:09:51,782", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-29 22:09:55,717", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 22:17:35,233", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-29 22:17:40,016", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 22:17:42,469", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-03-29 22:34:03,820", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-03-29 22:34:11,175", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-03-29 22:34:11,176", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-03-29 22:34:14,916", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-29 22:34:29,373", "level": "WARNING", "message": "Failed login attempt for jiyachaudhari0703@gmail.com from 127.0.0.1."}
{"time": "2025-03-29 22:34:29,389", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari0703@gmail.com from 127.0.0.1"}
{"time": "2025-03-29 22:34:29,395", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-29 22:35:05,534", "level": "WARNING", "message": "Failed login attempt for subham from 127.0.0.1."}
{"time": "2025-03-29 22:35:05,535", "level": "WARNING", "message": "Failed login attempt for username: subham from 127.0.0.1"}
{"time": "2025-03-29 22:35:05,541", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-03-29 22:35:17,604", "level": "INFO", "message": "User jiya logged in from 127.0.0.1."}
{"time": "2025-03-29 22:35:17,604", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-03-29 22:35:17,609", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-03-29 22:35:17,817", "level": "WARNING", "message": "Forbidden (CSRF token from POST incorrect.): /accounts/login/"}
{"time": "2025-03-29 22:35:17,824", "level": "WARNING", "message": ""POST /accounts/login/ HTTP/1.1" 403 2517"}
{"time": "2025-03-29 22:35:17,839", "level": "INFO", "message": "- Broken pipe from ('127.0.0.1', 62949)"}
{"time": "2025-04-01 08:47:14,909", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 08:47:39,101", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-04-01 08:47:39,116", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-04-01 08:47:39,189", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-04-01 08:47:39,190", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 2356"}
{"time": "2025-04-01 08:47:42,247", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-04-01 08:47:46,607", "level": "WARNING", "message": "Failed login attempt for jiysgdcskdg from 127.0.0.1."}
{"time": "2025-04-01 08:47:46,608", "level": "WARNING", "message": "Failed login attempt for username: jiysgdcskdg from 127.0.0.1"}
{"time": "2025-04-01 08:47:46,610", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-04-01 09:30:39,468", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 09:30:47,077", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-04-01 09:30:47,078", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-04-01 09:30:55,194", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-04-01 09:31:27,750", "level": "WARNING", "message": "Failed login attempt for jiyachaudhari0703@gmail.com from 127.0.0.1."}
{"time": "2025-04-01 09:31:27,750", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari0703@gmail.com from 127.0.0.1"}
{"time": "2025-04-01 09:31:27,767", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 648"}
{"time": "2025-04-01 09:48:54,615", "level": "WARNING", "message": "Failed login attempt for jiyachaudhari0703@gmail.com from 127.0.0.1."}
{"time": "2025-04-01 09:48:54,617", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari0703@gmail.com from 127.0.0.1"}
{"time": "2025-04-01 09:48:54,651", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 10:07:34,649", "level": "WARNING", "message": "Failed login attempt for jiyachaudhari0703@gmail.com from 127.0.0.1."}
{"time": "2025-04-01 10:07:34,649", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari0703@gmail.com from 127.0.0.1"}
{"time": "2025-04-01 10:07:34,669", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 10:08:05,153", "level": "INFO", "message": "User jiya logged in from 127.0.0.1."}
{"time": "2025-04-01 10:08:05,154", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-01 10:08:05,199", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-01 10:08:05,220", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-01 10:32:13,777", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-01 10:32:13,781", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-01 10:32:13,790", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 10:35:52,496", "level": "WARNING", "message": "Failed login attempt for jiyachaudhari0703@gmail.com from 127.0.0.1."}
{"time": "2025-04-01 10:35:52,496", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari0703@gmail.com from 127.0.0.1"}
{"time": "2025-04-01 10:35:52,521", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 10:43:56,338", "level": "INFO", "message": "User jiya logged in from 127.0.0.1."}
{"time": "2025-04-01 10:43:56,338", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-01 10:43:56,340", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-01 10:43:56,354", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-01 10:47:11,515", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 10:54:43,619", "level": "INFO", "message": "User jiya logged in from 127.0.0.1."}
{"time": "2025-04-01 10:54:43,619", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-01 10:54:43,624", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-01 10:54:43,630", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-01 10:54:51,623", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-01 10:54:51,623", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-01 10:54:51,629", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 10:54:59,089", "level": "WARNING", "message": "Failed login attempt for jiya from 127.0.0.1."}
{"time": "2025-04-01 10:54:59,111", "level": "WARNING", "message": "Failed login attempt for username: jiya from 127.0.0.1"}
{"time": "2025-04-01 10:54:59,111", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 10:55:25,190", "level": "WARNING", "message": "Failed login attempt for jiyachaudhari0703@gmail.com from 127.0.0.1."}
{"time": "2025-04-01 10:55:25,190", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari0703@gmail.com from 127.0.0.1"}
{"time": "2025-04-01 10:55:25,191", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 10:56:42,644", "level": "INFO", "message": "User jiya logged in from 127.0.0.1."}
{"time": "2025-04-01 10:56:42,644", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-01 10:56:42,666", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-01 10:56:42,673", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-01 10:56:49,126", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-01 10:56:49,128", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-01 10:56:49,133", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 10:56:54,962", "level": "WARNING", "message": "Failed login attempt for jiya from 127.0.0.1."}
{"time": "2025-04-01 10:56:54,962", "level": "WARNING", "message": "Failed login attempt for username: jiya from 127.0.0.1"}
{"time": "2025-04-01 10:56:54,962", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 11:03:31,065", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-04-01 11:03:35,452", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 11:05:39,460", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-04-01 11:05:47,028", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 11:05:59,561", "level": "WARNING", "message": "Failed login attempt for jiya from 127.0.0.1."}
{"time": "2025-04-01 11:05:59,561", "level": "WARNING", "message": "Failed login attempt for username: jiya from 127.0.0.1"}
{"time": "2025-04-01 11:05:59,957", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 11:06:09,674", "level": "WARNING", "message": "Failed login attempt for jiya from 127.0.0.1."}
{"time": "2025-04-01 11:06:09,674", "level": "WARNING", "message": "Failed login attempt for username: jiya from 127.0.0.1"}
{"time": "2025-04-01 11:06:09,859", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 11:30:53,510", "level": "WARNING", "message": "Failed login attempt for jiya from 127.0.0.1."}
{"time": "2025-04-01 11:30:53,515", "level": "WARNING", "message": "Failed login attempt for username: jiya from 127.0.0.1"}
{"time": "2025-04-01 11:30:53,840", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 11:35:20,452", "level": "WARNING", "message": "Failed login attempt for jiya from 127.0.0.1."}
{"time": "2025-04-01 11:35:20,474", "level": "WARNING", "message": "Failed login attempt for username: jiya from 127.0.0.1"}
{"time": "2025-04-01 11:35:20,977", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 11:38:58,038", "level": "WARNING", "message": "Failed login attempt for shreyachakraborty1712@gmail.com from 127.0.0.1."}
{"time": "2025-04-01 11:38:58,038", "level": "WARNING", "message": "Failed login attempt for username: shreyachakraborty1712@gmail.com from 127.0.0.1"}
{"time": "2025-04-01 11:38:58,399", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 11:41:12,135", "level": "WARNING", "message": "Failed login attempt for jiya from 127.0.0.1."}
{"time": "2025-04-01 11:41:12,642", "level": "WARNING", "message": "Failed login attempt for username: jiya from 127.0.0.1"}
{"time": "2025-04-01 11:41:12,951", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 11:43:56,682", "level": "WARNING", "message": "Failed login attempt for jiya from 127.0.0.1."}
{"time": "2025-04-01 11:43:57,023", "level": "WARNING", "message": "Failed login attempt for username: jiya from 127.0.0.1"}
{"time": "2025-04-01 11:43:57,181", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 12:06:45,495", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-04-01 12:06:49,456", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 12:10:17,211", "level": "WARNING", "message": "Failed login attempt for jiyachaudhari0703@gmail.com from 127.0.0.1."}
{"time": "2025-04-01 12:10:17,240", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari0703@gmail.com from 127.0.0.1"}
{"time": "2025-04-01 12:10:17,881", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 968"}
{"time": "2025-04-01 12:47:58,479", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 12:49:17,732", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-04-01 12:49:19,949", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 12:49:52,214", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 12:49:57,009", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-04-01 12:49:57,011", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-04-01 12:50:02,041", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-01 12:50:18,037", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari"}
{"time": "2025-04-01 12:50:18,433", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-01 12:50:40,185", "level": "INFO", "message": ""GET /accounts/register HTTP/1.1" 301 0"}
{"time": "2025-04-01 12:50:40,500", "level": "ERROR", "message": "Internal Server Error: /accounts/register/"}
Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py", line 56, in register
    return render(request, 'register.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: register.html
{"time": "2025-04-01 12:50:40,516", "level": "ERROR", "message": ""GET /accounts/register/ HTTP/1.1" 500 86965"}
{"time": "2025-04-01 12:50:54,303", "level": "ERROR", "message": "Internal Server Error: /accounts/register/"}
Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py", line 56, in register
    return render(request, 'register.html')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\shortcuts.py", line 25, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: register.html
{"time": "2025-04-01 12:50:54,303", "level": "ERROR", "message": ""GET /accounts/register/ HTTP/1.1" 500 86965"}
{"time": "2025-04-01 12:58:43,406", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:12:40,355", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:13:50,867", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\cosmos.py changed, reloading."}
{"time": "2025-04-01 15:13:53,823", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:14:32,304", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\signals.py changed, reloading."}
{"time": "2025-04-01 15:14:35,214", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:14:42,377", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\signals.py changed, reloading."}
{"time": "2025-04-01 15:14:44,951", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:15:33,970", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-04-01 15:15:36,027", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:16:32,726", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-04-01 15:16:34,215", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:17:24,992", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-04-01 15:17:27,025", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:17:55,171", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-04-01 15:17:57,493", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:19:04,530", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\urls.py changed, reloading."}
{"time": "2025-04-01 15:19:06,482", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:19:10,341", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\signals.py changed, reloading."}
{"time": "2025-04-01 15:19:12,993", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:19:45,806", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:20:42,347", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\urls.py changed, reloading."}
{"time": "2025-04-01 15:20:44,303", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:20:49,504", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\urls.py changed, reloading."}
{"time": "2025-04-01 15:20:51,996", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:21:35,438", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 15:21:40,148", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-04-01 15:21:40,149", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-04-01 15:21:40,257", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-04-01 15:21:40,260", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 2356"}
{"time": "2025-04-01 17:54:02,851", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-01 17:54:13,329", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-04-01 17:54:13,329", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-04-01 17:54:13,440", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-04-01 17:54:13,441", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 2356"}
{"time": "2025-04-01 17:54:21,410", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-01 17:54:32,740", "level": "WARNING", "message": "Failed login attempt for username: jiya"}
{"time": "2025-04-01 17:54:33,082", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 09:01:52,365", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-03 09:04:45,057", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\signals.py changed, reloading."}
{"time": "2025-04-03 09:05:02,281", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-03 09:10:41,730", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-03 09:15:48,348", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\utils.py changed, reloading."}
{"time": "2025-04-03 09:15:48,347", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\utils.py changed, reloading."}
{"time": "2025-04-03 09:15:55,110", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-03 09:15:55,609", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-03 09:17:48,353", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-04-03 09:17:48,789", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-04-03 09:17:50,903", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-03 09:17:51,066", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-03 09:33:51,478", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-04-03 09:33:51,795", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-04-03 09:33:59,925", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-03 09:33:59,925", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-03 09:34:01,735", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-03 09:34:12,054", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-04-03 09:34:12,055", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-04-03 09:34:12,121", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-04-03 09:34:12,122", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 2356"}
{"time": "2025-04-03 09:34:16,065", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 09:34:34,862", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari0703@gmail.com"}
{"time": "2025-04-03 09:34:35,196", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 09:39:33,816", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari0703@gmail.com"}
{"time": "2025-04-03 09:39:34,021", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 09:46:24,013", "level": "WARNING", "message": "Failed login attempt for username: shreyachakraborty1712@gmail.com"}
{"time": "2025-04-03 09:46:24,254", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 09:56:12,340", "level": "WARNING", "message": "Failed login attempt for username: shreyachakraborty1712@gmail.com"}
{"time": "2025-04-03 09:56:12,541", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 09:57:22,278", "level": "WARNING", "message": "Failed login attempt for username: shreyachakraborty1712@gmail.com"}
{"time": "2025-04-03 09:57:22,471", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 09:57:42,859", "level": "WARNING", "message": "Failed login attempt for username: ruchirchitre@gmail.com"}
{"time": "2025-04-03 09:57:43,019", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:06:11,215", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-04-03 10:06:11,234", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-04-03 10:06:11,234", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\mywebsite\settings.py changed, reloading."}
{"time": "2025-04-03 10:06:14,125", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-03 10:06:14,125", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-03 10:24:02,945", "level": "WARNING", "message": "Failed login attempt for username: khghkj.com"}
{"time": "2025-04-03 10:24:03,305", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:25:12,385", "level": "WARNING", "message": "Failed login attempt for username: jiya.com"}
{"time": "2025-04-03 10:25:12,616", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:26:36,850", "level": "WARNING", "message": "Failed login attempt for username: shreyaaaa.com"}
{"time": "2025-04-03 10:26:37,116", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:29:06,392", "level": "WARNING", "message": "Failed login attempt for username: shreya.com"}
{"time": "2025-04-03 10:29:06,700", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:30:11,526", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari0703@gmail.com"}
{"time": "2025-04-03 10:30:11,833", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:30:42,033", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 10:30:42,389", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:30:42,407", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 10:31:26,044", "level": "WARNING", "message": "Forbidden (Permission denied): /admin/auth/user/"}
Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 715, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\sites.py", line 240, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 1981, in changelist_view
    raise PermissionDenied
django.core.exceptions.PermissionDenied
{"time": "2025-04-03 10:31:26,051", "level": "WARNING", "message": ""GET /admin/auth/user/ HTTP/1.1" 403 135"}
{"time": "2025-04-03 10:31:31,362", "level": "INFO", "message": ""GET /admin HTTP/1.1" 301 0"}
{"time": "2025-04-03 10:31:31,694", "level": "INFO", "message": ""GET /admin/ HTTP/1.1" 200 4503"}
{"time": "2025-04-03 10:31:32,410", "level": "INFO", "message": ""GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2810"}
{"time": "2025-04-03 10:31:32,410", "level": "INFO", "message": ""GET /static/admin/css/responsive.css HTTP/1.1" 200 17905"}
{"time": "2025-04-03 10:31:32,415", "level": "INFO", "message": ""GET /static/admin/js/theme.js HTTP/1.1" 200 1943"}
{"time": "2025-04-03 10:31:32,428", "level": "INFO", "message": ""GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2682"}
{"time": "2025-04-03 10:31:32,431", "level": "INFO", "message": ""GET /static/admin/css/dashboard.css HTTP/1.1" 200 441"}
{"time": "2025-04-03 10:31:32,436", "level": "INFO", "message": ""GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063"}
{"time": "2025-04-03 10:31:32,437", "level": "INFO", "message": ""GET /static/admin/css/base.css HTTP/1.1" 200 21544"}
{"time": "2025-04-03 10:31:39,832", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-04-03 10:31:39,832", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-04-03 10:31:46,624", "level": "WARNING", "message": "Forbidden (Permission denied): /admin/auth/user/"}
Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 715, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\sites.py", line 240, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 1981, in changelist_view
    raise PermissionDenied
django.core.exceptions.PermissionDenied
{"time": "2025-04-03 10:31:46,630", "level": "WARNING", "message": ""GET /admin/auth/user/ HTTP/1.1" 403 135"}
{"time": "2025-04-03 10:31:51,383", "level": "INFO", "message": ""GET /admin/ HTTP/1.1" 200 4503"}
{"time": "2025-04-03 10:31:59,845", "level": "INFO", "message": ""POST /admin/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:31:59,851", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:39:03,889", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:39:04,120", "level": "INFO", "message": ""GET /admin/login/?next=/admin/auth/user/ HTTP/1.1" 200 4178"}
{"time": "2025-04-03 10:39:04,150", "level": "INFO", "message": ""GET /static/admin/css/login.css HTTP/1.1" 200 958"}
{"time": "2025-04-03 10:39:20,599", "level": "INFO", "message": ""POST /admin/login/?next=/admin/auth/user/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:39:20,618", "level": "WARNING", "message": "Forbidden (Permission denied): /admin/auth/user/"}
Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 715, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\sites.py", line 240, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 1981, in changelist_view
    raise PermissionDenied
django.core.exceptions.PermissionDenied
{"time": "2025-04-03 10:39:20,622", "level": "WARNING", "message": ""GET /admin/auth/user/ HTTP/1.1" 403 135"}
{"time": "2025-04-03 10:39:25,164", "level": "INFO", "message": ""GET /admin/login/?next=/admin/auth/user/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:39:25,178", "level": "INFO", "message": ""GET /admin/ HTTP/1.1" 200 4503"}
{"time": "2025-04-03 10:39:28,073", "level": "INFO", "message": ""POST /admin/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:39:28,078", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:39:36,030", "level": "INFO", "message": ""GET /admin/auth/user/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:39:36,046", "level": "INFO", "message": ""GET /admin/login/?next=/admin/auth/user/ HTTP/1.1" 200 4178"}
{"time": "2025-04-03 10:39:42,420", "level": "INFO", "message": ""POST /admin/login/?next=/admin/auth/user/ HTTP/1.1" 200 4361"}
{"time": "2025-04-03 10:39:58,250", "level": "INFO", "message": ""POST /admin/login/?next=/admin/auth/user/ HTTP/1.1" 200 4361"}
{"time": "2025-04-03 10:40:08,037", "level": "INFO", "message": ""POST /admin/login/?next=/admin/auth/user/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:40:08,051", "level": "WARNING", "message": "Forbidden (Permission denied): /admin/auth/user/"}
Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 715, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\views\decorators\cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\sites.py", line 240, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\contrib\admin\options.py", line 1981, in changelist_view
    raise PermissionDenied
django.core.exceptions.PermissionDenied
{"time": "2025-04-03 10:40:08,051", "level": "WARNING", "message": ""GET /admin/auth/user/ HTTP/1.1" 403 135"}
{"time": "2025-04-03 10:41:12,293", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-04-03 10:41:12,293", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-04-03 10:41:26,109", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:42:21,817", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 10:42:22,059", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:42:22,086", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 10:43:17,170", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 10:43:17,174", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:43:17,177", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:43:37,923", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 10:43:38,166", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:43:38,173", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 10:43:42,576", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 10:43:42,577", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:43:42,582", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:43:52,916", "level": "WARNING", "message": "Failed login attempt for username: jiya.com"}
{"time": "2025-04-03 10:43:53,184", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:45:53,505", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 10:45:53,737", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:45:53,749", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 10:50:30,287", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 10:50:30,291", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:50:30,294", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:50:40,018", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 10:50:40,433", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:50:40,445", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 10:51:07,658", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 10:51:07,660", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:51:07,665", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:51:15,582", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 10:51:15,868", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:51:15,881", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 10:52:11,245", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 10:52:11,245", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:52:11,252", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:52:23,962", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 10:52:24,597", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:52:24,609", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 10:57:11,609", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 10:57:11,610", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:57:11,613", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:57:18,040", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 10:57:18,282", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:57:18,296", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 10:57:48,910", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 10:57:48,911", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:57:48,919", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:57:56,440", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 10:57:56,666", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:57:56,677", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 10:58:12,891", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 10:58:12,893", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 10:58:12,896", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 10:58:34,018", "level": "WARNING", "message": "Failed login attempt for username: jiya.com"}
{"time": "2025-04-03 10:58:34,342", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:00:26,160", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 11:00:26,502", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 11:00:26,513", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 11:00:42,467", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 11:00:42,472", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 11:00:42,475", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:00:54,768", "level": "WARNING", "message": "Failed login attempt for username: jfjmkhjgjk"}
{"time": "2025-04-03 11:00:55,137", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:02:23,973", "level": "WARNING", "message": "Failed login attempt for username: dsxgdh"}
{"time": "2025-04-03 11:02:24,254", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:02:50,081", "level": "WARNING", "message": "Failed login attempt for username: jyfmhm"}
{"time": "2025-04-03 11:02:50,556", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:03:32,730", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 11:03:32,986", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 11:03:32,998", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 11:04:20,098", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 11:04:20,101", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 11:04:20,101", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:04:32,742", "level": "WARNING", "message": "Failed login attempt for username: xhw.com"}
{"time": "2025-04-03 11:04:33,049", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:08:16,497", "level": "WARNING", "message": "Failed login attempt for username: jiya.com"}
{"time": "2025-04-03 11:08:16,906", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:08:39,429", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 11:08:39,972", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 11:08:39,990", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 11:09:33,316", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 11:09:33,318", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 11:09:33,320", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:09:43,016", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 11:09:43,331", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 11:09:43,346", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 11:10:47,219", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 11:10:47,221", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 11:10:47,226", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:10:53,969", "level": "WARNING", "message": "Failed login attempt for username: JIYA"}
{"time": "2025-04-03 11:10:54,286", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:11:49,176", "level": "WARNING", "message": "Failed login attempt for username: jiiya"}
{"time": "2025-04-03 11:11:49,382", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:13:18,236", "level": "WARNING", "message": "Failed login attempt for username: JIYA"}
{"time": "2025-04-03 11:13:18,572", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:13:32,506", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 11:13:32,700", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 11:13:32,715", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 11:14:56,735", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 11:14:56,738", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 11:14:56,740", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 11:15:03,933", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 11:15:04,114", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 11:15:04,128", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 12:08:07,961", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-03 12:08:16,710", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-04-03 12:08:16,710", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-04-03 12:08:17,033", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-04-03 12:08:17,033", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 2356"}
{"time": "2025-04-03 12:08:28,817", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 12:08:53,163", "level": "ERROR", "message": "Error sending data to Logic App: HTTPSConnectionPool(host='prod-15.centralindia.logic.azure.com', port=443): Max retries exceeded with url: /workflows/577d0fc584004453b6a0c23e17543319/triggers/When_a_HTTP_request_is_received/paths/invoke?api-version=2016-10-01&sp=%2Ftriggers%2FWhen_a_HTTP_request_is_received%2Frun&sv=1.0&sig=hWGozbqDblcjs7SGCikpryujYZAuf3BkfGkCa-KkTtg (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000021531C99450>: Failed to resolve 'prod-15.centralindia.logic.azure.com' ([Errno 11001] getaddrinfo failed)"))"}
{"time": "2025-04-03 12:08:53,164", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 12:08:53,166", "level": "ERROR", "message": "Error sending login data to Logic App: HTTPSConnectionPool(host='prod-15.centralindia.logic.azure.com', port=443): Max retries exceeded with url: /workflows/577d0fc584004453b6a0c23e17543319/triggers/When_a_HTTP_request_is_received/paths/invoke?api-version=2016-10-01&sp=%2Ftriggers%2FWhen_a_HTTP_request_is_received%2Frun&sv=1.0&sig=hWGozbqDblcjs7SGCikpryujYZAuf3BkfGkCa-KkTtg (Caused by NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x0000021531CD7C90>: Failed to resolve 'prod-15.centralindia.logic.azure.com' ([Errno 11001] getaddrinfo failed)"))"}
{"time": "2025-04-03 12:08:53,178", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 12:08:53,186", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-03 12:14:08,219", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-03 12:14:08,221", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 12:14:08,222", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 12:14:14,758", "level": "WARNING", "message": "Failed login attempt for username: clkvsfhvnl.com"}
{"time": "2025-04-03 12:14:15,134", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-03 12:14:54,674", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-03 12:14:54,927", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-03 12:14:54,936", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-05 21:29:26,347", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-05 21:29:35,583", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-04-05 21:29:35,584", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-04-05 21:29:35,649", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-04-05 21:29:35,649", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 2356"}
{"time": "2025-04-05 21:29:39,305", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-05 21:29:55,355", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-05 21:29:55,507", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-05 21:29:55,522", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-05 21:30:14,915", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-05 21:30:14,916", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-05 21:30:14,920", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-05 21:30:30,521", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari"}
{"time": "2025-04-05 21:30:31,069", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-07 11:36:42,564", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-07 11:36:51,798", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-04-07 11:36:51,798", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-04-07 11:36:51,933", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-04-07 11:36:51,935", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 2356"}
{"time": "2025-04-07 11:36:54,804", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-07 11:37:08,739", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-07 11:37:10,092", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-07 11:37:10,138", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-07 11:47:31,627", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-07 11:47:31,629", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-07 11:47:31,633", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-07 11:47:43,988", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari"}
{"time": "2025-04-07 11:47:44,977", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-07 12:04:42,628", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-07 12:04:43,151", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-07 12:04:43,159", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-07 12:05:05,782", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-07 12:05:05,784", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-07 12:05:05,789", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-07 12:05:13,079", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari"}
{"time": "2025-04-07 12:05:13,694", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-07 13:01:09,834", "level": "ERROR", "message": "Invalid HTTP_HOST header: '8429-2402-3a80-4299-e96e-799a-1b44-4c9f-302c.ngrok-free.app'. You may need to add '8429-2402-3a80-4299-e96e-799a-1b44-4c9f-302c.ngrok-free.app' to ALLOWED_HOSTS."}
Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\http\request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8429-2402-3a80-4299-e96e-799a-1b44-4c9f-302c.ngrok-free.app'. You may need to add '8429-2402-3a80-4299-e96e-799a-1b44-4c9f-302c.ngrok-free.app' to ALLOWED_HOSTS.
{"time": "2025-04-07 13:01:10,089", "level": "WARNING", "message": "Bad Request: /"}
{"time": "2025-04-07 13:01:10,092", "level": "WARNING", "message": ""GET / HTTP/1.1" 400 76691"}
{"time": "2025-04-07 13:01:11,054", "level": "ERROR", "message": "Invalid HTTP_HOST header: '8429-2402-3a80-4299-e96e-799a-1b44-4c9f-302c.ngrok-free.app'. You may need to add '8429-2402-3a80-4299-e96e-799a-1b44-4c9f-302c.ngrok-free.app' to ALLOWED_HOSTS."}
Traceback (most recent call last):
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\utils\deprecation.py", line 133, in __call__
    response = self.process_request(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\middleware\common.py", line 48, in process_request
    host = request.get_host()
           ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Subham\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\django\http\request.py", line 151, in get_host
    raise DisallowedHost(msg)
django.core.exceptions.DisallowedHost: Invalid HTTP_HOST header: '8429-2402-3a80-4299-e96e-799a-1b44-4c9f-302c.ngrok-free.app'. You may need to add '8429-2402-3a80-4299-e96e-799a-1b44-4c9f-302c.ngrok-free.app' to ALLOWED_HOSTS.
{"time": "2025-04-07 13:01:11,103", "level": "WARNING", "message": "Bad Request: /favicon.ico"}
{"time": "2025-04-07 13:01:11,105", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 400 76518"}
{"time": "2025-04-08 21:10:55,610", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-08 21:11:32,494", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-04-08 21:11:32,494", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-04-08 21:11:32,571", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-04-08 21:11:32,571", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 2356"}
{"time": "2025-04-08 21:11:35,712", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-08 21:11:39,909", "level": "WARNING", "message": "{"Username": "jiya", "Status": "Failed", "IP": "127.0.0.1", "Timestamp": "2025-04-08T15:41:39.909159+00:00"}"}
{"time": "2025-04-08 21:11:40,605", "level": "WARNING", "message": "Failed login attempt for username: jiya"}
{"time": "2025-04-08 21:11:40,884", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-08 21:11:55,073", "level": "WARNING", "message": "{"Username": "jiyachaudhari", "Status": "Failed", "IP": "127.0.0.1", "Timestamp": "2025-04-08T15:41:55.073987+00:00"}"}
{"time": "2025-04-08 21:11:55,077", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari"}
{"time": "2025-04-08 21:11:55,429", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-08 21:24:00,960", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\signals.py changed, reloading."}
{"time": "2025-04-08 21:24:04,678", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-08 21:28:18,377", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\views.py changed, reloading."}
{"time": "2025-04-08 21:28:20,782", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-08 21:29:27,022", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\__init__.py changed, reloading."}
{"time": "2025-04-08 21:29:29,392", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-08 21:29:33,820", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\__init__.py changed, reloading."}
{"time": "2025-04-08 21:29:37,174", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-08 21:32:44,693", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\signals.py changed, reloading."}
{"time": "2025-04-08 21:32:47,761", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-08 21:32:55,589", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\signals.py changed, reloading."}
{"time": "2025-04-08 21:32:59,307", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-08 21:34:14,997", "level": "INFO", "message": "C:\Users\Subham\Desktop\CC\mywebsite\accounts\signals.py changed, reloading."}
{"time": "2025-04-08 21:34:20,069", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-09 12:04:25,229", "level": "INFO", "message": "Watching for file changes with StatReloader"}
{"time": "2025-04-09 12:04:32,904", "level": "WARNING", "message": "Not Found: /"}
{"time": "2025-04-09 12:04:32,906", "level": "WARNING", "message": ""GET / HTTP/1.1" 404 2305"}
{"time": "2025-04-09 12:04:33,111", "level": "WARNING", "message": "Not Found: /favicon.ico"}
{"time": "2025-04-09 12:04:33,114", "level": "WARNING", "message": ""GET /favicon.ico HTTP/1.1" 404 2356"}
{"time": "2025-04-09 12:04:37,706", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-09 12:14:07,959", "level": "INFO", "message": "{"Username": "jiya", "Status": "Success", "IP": "127.0.0.1", "Timestamp": "2025-04-09T06:44:07.959450+00:00"}"}
{"time": "2025-04-09 12:14:08,504", "level": "INFO", "message": "User jiya logged in successfully from 127.0.0.1."}
{"time": "2025-04-09 12:14:08,953", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 302 0"}
{"time": "2025-04-09 12:14:08,989", "level": "INFO", "message": ""GET /accounts/home/ HTTP/1.1" 200 257"}
{"time": "2025-04-09 12:14:16,526", "level": "INFO", "message": "{"Username": "jiya", "Status": "Logout", "IP": "127.0.0.1", "Timestamp": "2025-04-09T06:44:16.526025+00:00"}"}
{"time": "2025-04-09 12:14:16,538", "level": "INFO", "message": "User jiya logged out successfully from 127.0.0.1."}
{"time": "2025-04-09 12:14:16,540", "level": "INFO", "message": ""GET /accounts/logout/ HTTP/1.1" 302 0"}
{"time": "2025-04-09 12:14:16,546", "level": "INFO", "message": ""GET /accounts/login/ HTTP/1.1" 200 735"}
{"time": "2025-04-09 12:14:25,605", "level": "WARNING", "message": "Failed login attempt for username: jiyachaudhari"}
{"time": "2025-04-09 12:14:26,014", "level": "INFO", "message": ""POST /accounts/login/ HTTP/1.1" 200 735"}
